dask/dataframe/tests/test_dataframe.py

```diff
@@ -907,6 +907,11 @@ def test_map_partitions():
     assert_eq(d.map_partitions(lambda df: df, meta=d), full)
     assert_eq(d.map_partitions(lambda df: df), full)
     result = d.map_partitions(lambda df: df.sum(axis=1))
+    layer = [
+        layer for k, layer in result.dask.layers.items() if k.startswith("lambda-")
```

Comment:
TODO use `hlg_layer` helpers in https://github.com/dask/dask/pull/8341 once that's merged